src/pypromice/process/aws.py

```diff
@@ -91,6 +91,7 @@ def getL1(self):
         logger.info('Level 1 processing...')
         self.L0 = [utilities.addBasicMeta(item, self.vars) for item in self.L0]
         self.L1 = [toL1(item, self.vars) for item in self.L0]
+        self.L1.reverse()
```

Comment:
For info, I did that because of attribute management: the attributes item in the list overwrites the values of all other items for a given key. For station changing from one boom two two booms (EGP, KAN_U), we need to make sure this list is properly ordered.